features/access_management.md

```diff
@@ -0,0 +1,24 @@
+# Access management
+TokenD is based on a permissioned blockchain that limit the parties (it can be wether another person or external service) who can transact on the blockchain and that set who can serve the network by writing new blocks into the chain.
+
+Therefore Tokend has pack of functionalities that help us to distribute the rights. In order to use them, you need to be familiar with 2 key entities: **accounts** and **signers**
+
+### Accounts
+
+Accounts are the central data structure in TokenD. First of all, an account is a human-readable name that is stored on the blockchain. An account is required to transfer or otherwise push a transaction to the blockchain. Accounts are identified by a public key and saved in the ledger. Another data in the ledger, such as assets, offers, or KYC data are owned by a particular account.
+
+TokenD differs the type of account to master's and user's [accounts](/tech/key_entities/accounts#account-type). The owner of master account is allowed to perform a lot of operations that will have impact on the whole system.

```

Comment:
In TokenD, we differ common user accounts (see [their types][link here]) that can manage only themselves from the account that can manage the whole system ([master][link] account). By being the owner of master account, you are allowed to perform a set of operations that will impact the system. (For example, you can set fees, review KYC requests, manage secondary markets etc.)